plugin/src/main/java/com/denizenscript/denizen/objects/properties/entity/EntityColor.java

```diff
@@ -92,7 +92,26 @@ public void setPropertyValue(ElementTag color, Mechanism mechanism) {
         else if (type == EntityType.SHEEP && mechanism.requireEnum(DyeColor.class)) {
             as(Sheep.class).setColor(color.asEnum(DyeColor.class));
         }
-        else if (type == EntityType.WOLF && mechanism.requireEnum(DyeColor.class)) {
+        else if (type == EntityType.WOLF && NMSHandler.getVersion().isAtLeast(NMSVersion.v1_20) && mechanism.requireObject(ListTag.class)) {

```

Comment:
iirc require list does nothing

Reply:
... all the other code here runs that actually so eh matching the pattern is fine actually nvm